prov/verbs/src/verbs_eq.c

```diff
@@ -73,14 +73,10 @@ static struct fi_info *
 fi_ibv_eq_cm_getinfo(struct fi_ibv_fabric *fab, struct rdma_cm_event *event,
 		struct fi_info *pep_info)
 {
-	struct fi_info *info, *fi;
+	struct fi_info *info;
 	struct fi_ibv_connreq *connreq;
 
-	fi = fi_ibv_get_verbs_info(ibv_get_device_name(event->id->verbs->device));
-	if (!fi)
-		return NULL;
-
-	info = fi_dupinfo(fi);
+	info = fi_dupinfo(fab->info);
```

Comment:
>  Is it possible to get a connection request for a fabric that is not in fab->info ? 

Yes. If the server listens on a wildcard address, the fi_info passed by the user in fi_fabric call may not be the same as one where a connection request is received. If we reject such requests, listening on wildcard address won't be possible. We would like to have the capability to listen on wildcard address.